php-practice_2.php

```diff
@@ -0,0 +1,116 @@
+<?php
+// Q1 tic-tac問題
+
+for ($i=1; $i<=100; ++$i) {
+    echo PHP_EOL;
+if ($i % 4 == 0 || $i % 5 == 0){
+    if ($i % 4 == 0 && $i % 5 == 0) {
+        echo 'tic-tac';
+    } elseif (($i % 4) == 0){
+        echo 'tic';
+    } elseif (($i % 5) == 0) {
+        echo 'tac';
+    } 
+}  else {
+    echo $i;
+}
+}
```

Comment:
これでもいけますね！


ifの中にifが書いていないぶん可読性が高いです！
```suggestion
    if ($i % 4 == 0 && $i % 5 == 0) {
        echo 'tic-tac';
    } elseif (($i % 4) == 0){
        echo 'tic';
    } elseif (($i % 5) == 0) {
        echo 'tac';
    } else {
    echo $i;
}

```

Reply:
修正しました。